be/src/exec/pipeline/pipeline_driver_queue.cpp

```diff
@@ -309,6 +314,11 @@ void WorkGroupDriverQueue::cancel(DriverRawPtr driver) {
 }
 
 void WorkGroupDriverQueue::update_statistics(const DriverRawPtr driver) {
+    // we don't have to update statistics when we only have one work group
+    if (ExecEnv::GetInstance()->workgroup_manager()->num_workgroups() <= 1) {
+        return;
```

Comment:
Maybe we still need update statistics of `wg_entity->queue()? when we only have one work group, the reason is that `wg_entity->queue()` is the inner multiple level feedback queue for this single workgroup, need update its statistics for multiple level feedback strategy.

Reply:
ok, I'll try to update only the inner-queue stats only when we only have one wg.